desktop/apps/artwork/client/index.coffee

```diff
@@ -72,9 +73,20 @@ module.exports =
           require('../../../components/current_auctions/index.jsx').default unless context.is_closed
           require '../components/artist_artworks/index.coffee' if context.is_closed
           require '../components/related_artworks/index.coffee' if context.is_closed
-          require '../components/related_artists/index.coffee'
+          require '../components/related_artists/index.coffee',
         ]
 
+      # Make some Auctions-only layout tweaks. See https://github.com/artsy/auctions/issues/360#event-1017559326
+      # for more info.
+      postInit: ->
+        $currentAuctions = $ '.artwork-current-auctions'
+        $relatedRail = $ '.artwork-artist-related-rail'
+        paddingTop = if $currentAuctions.length then '42px' else '31px'
+
+        $relatedRail
+          .css('border-top', 0)
+          .css('padding-top', paddingTop)
```

Comment:
I can drop a style rule into the `current_auctions` stylus module that targets a totally separate component (`.artwork-artist-related-rail`), so that when `current_auctions` mounts the selector becomes active and updates, but this is wild indirection compared to a declarative `postInit` function that toggles two CSS props, with a comment pointing to a GH issue describing the general deviation from the overall layout. 

I'm in complete agreement that adjusting CSS with JS can grow as a hack and one should be cautious, but the ask is fairly straightforward and clear within a fairly complex design with many different states.